microhapulator/cli/pipe.py

```diff
@@ -131,27 +131,31 @@ def subparser(subparsers):
         default=cpu_count(),
         help="process each batch using T threads; by default, one thread per available core is used",
     )
+    # Normally the defaults for the next three arguments would be `None`. But the values are passed
+    # to Snakemake, which intermittently casts NoneTypes as a string, causing issues with the
+    # downstream code; e.g., attempts to open a non-existent `None` file. The empty string defaults
+    # and the lambda functions for data types are a workaround.
     cli.add_argument(
         "-s",
         "--static",
         metavar="ST",
-        type=int,
-        default=None,
+        type=lambda a: "" if a == "" else int(a),
+        default="",
```

Comment:
May be of interest to you, @RyanBerger98. Seems related to issues we've seen recently with other Snakemake workflows.